src/python/runner/Benchmark.py

```diff
@@ -0,0 +1,85 @@
+import time, sys
+from collections import namedtuple
+
+from runner.ModuleLoader import module_load
+from shared import GMMData
+from shared import BAData
+from shared import HandData
+from shared import LSTMData
+from runner.Filepaths import filepath_to_basename, modulepath_to_basename
+from shared.output_utils import save_time_to_file
+
+measurable_time_not_achieved = -1
+
+def find_repeats_for_minimum_measurable_time(minimum_measurable_time, func):
+    total_time = 0
+    min_sample = sys.float_info.max
+
+    repeats = 1
+    max_possible_power_of_two = (sys.maxsize >> 1) + 1
+    while True:
+        t1 = time.time()
+        func(repeats)
+        t2 = time.time()
+        # Time in seconds
+        current_run_time = t2 - t1
+        if current_run_time > minimum_measurable_time:
+            current_sample = current_run_time / repeats
+            min_sample = min(min_sample, current_sample)
+            total_time += current_run_time
+            break
+        # The next iteration will overflow a loop counter that's why we recognize that we cannot reach the minimum measurable time.
+        if repeats == max_possible_power_of_two:
+            repeats = measurable_time_not_achieved
+            break
+        repeats *= 2
+        # loop exit condition
+        if repeats > max_possible_power_of_two:
```

Comment:
This condition seems unachievable, because of exit condition on line 32.

Reply:
I agree with this, exit condition simplified.